operator/apis/loki/v1/lokistack_types.go

```diff
@@ -269,6 +269,13 @@ type LokiComponentSpec struct {
 	// +optional
 	// +kubebuilder:validation:Optional
 	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
+
+	// PodAntiAffinity defines the pod anti affinity scheduling rules to schedule pods
+	// of a component.
+	//
+	// +optional
+	// +kubebuilder:validation:Optional
+	PodAntiAffinity *corev1.PodAntiAffinity `json:"podAntiAffinity,omitempty"`
```

Comment:
AFAIK, anti-affinity is only a pod-level concept in k8s. So WDYT to skip the `Pod` prefix here?

Reply:
I don't 100% agree with this just because it assumes the user also knows about that "truth", while the full name makes it more explicit and clear. But I'm happy to oblige if it makes more sense to drop the `Pod` prefix